tests/api_resources/test_file_upload.py

```diff
@@ -3,40 +3,65 @@
 import tempfile
 
 import stripe
-from tests.helper import StripeResourceTest
+from tests.helper import StripeMockTestCase
 
 
-class FileUploadTest(StripeResourceTest):
+TEST_RESOURCE_ID = 'file_123'
 
-    def test_create_file_upload(self):
-        test_file = tempfile.TemporaryFile()
-        stripe.FileUpload.create(
-            purpose='dispute_evidence',
-            file=test_file
-        )
-        self.requestor_mock.request.assert_called_with(
-            'post',
+
+class FileUploadTest(StripeMockTestCase):
+    FIXTURE = {
+        'id': TEST_RESOURCE_ID,
+        'object': 'file_upload'
+    }
+
+    def test_is_listable(self):
+        self.stub_request(
+            'get',
```

Comment:
is it safe to stub something like this? Feels like we're not testing the feature properly anymore

Reply:
Listing and retrieving file_upload objects is not different from retrieving any other API resources, save for the different hostname. I can add an assertion for the hostname if you think that's safer.

Creating a file_upload object is a bit different because of the `multipart` encoding, but most of the logic is tested in `test_multipart_data_generator.py`.